packages/odyssey-storybook/src/components/Button/Button.mdx

```diff
@@ -0,0 +1,187 @@
+
+**STORY MISSING**
+
+### Full-width
+
+Full-width Buttons are intended for single-interaction UIs. These are often widgets like "Sign In" or "Sign Up".
+
+Use this variant when you desire the Button to be full-width regardless of screen size.
+
+**STORY MISSING**
+
+## States
+
+Odyssey provides visual affordances these states: Enabled, Focus, Hover, and Disabled.
+
+### Enabled
+
+Buttons in their "normal" state are considered enabled. They are ready for user interaction.
+
+**STORY MISSING**
+
+### Focus
+
+The focus state is a visual affordance that the user has highlighted the Button with a pointer, keyboard, or voice.
+
+**STORY MISSING**
+
+### Hover
+
+Hover states are activated when the user pauses their pointer over the Button.
+
+**STORY MISSING**
+
+### Disabled
+
+Disabled Buttons are unavailable for interaction and cannot be focused. They can be used when an action is unavailable, possibly due to incomplete information or access restrictions.
+
+You should pair Disabled Buttons with a Tooltip if the user would benefit from additional context.
+
+**STORY MISSING**
+
+## Usage
+
+Use Buttons to indicate the in-page actions a user can take.
+
+**STORY MISSING**
+
+Don't use Buttons to navigate users around the site or product; use links instead.
+
+**STORY MISSING**
+
+## Content Guidelines
+
+Consider the following when writing content for Buttons. By doing so, it will ensure users can easily navigate the page and complete the task at hand efficiently.
+
+### Context
+
+Provide enough context to ensure the user is aware what the interaction will achieve. Avoid patterns that require the user to discover what a Button does.
+
+**STORY MISSING**
+
+Be strategic in your Button placement so a user has the best context. For example, consider using Buttons at the end of a form or preceding them with supporting copy.
+
+**STORY MISSING**
+
+### Word count
+
+Don't use more than three words inside of a Button. If a user needs more context, consider other design solutions.
+
+**STORY MISSING**
+
+**STORY MISSING**
+
+### Capitalization
+
+When using multiple words, use sentence case. Capitalize only the first letter and any proper nouns.
+
+**STORY MISSING**
+
+**STORY MISSING**
+
+### Buttons with Icons
+
+Icons can be added to any of our Button variants to increase clarity or add flair. Icons will be laid out automatically based on language direction.
+
+**STORY MISSING**
+
+#### Icon-only usability
+
+We recommend pairing icon-only Buttons with our Tooltip. While this is not required, it will increase clarity for sighted users.
+
+**STORY MISSING**
+
+## Accessibility
+
+Color is not a clear affordance for all users, please use clear, concise copy to label Buttons. Good rules of thumb: use three or less words to describe your action and start your label with a verb (e.g. "Access report" vs "Report PDF").
+
+## References
+
+### Related components
+
+- <a href="/components/link">Link</a>
+- <a href="/components/tooltip">Tooltip</a>
```

Comment:
these links (and any of them using a similar URL structure) will need to be fixed. The correct paths look like this:

```
/?path=/docs/components-link
```